users/models.py

```diff
@@ -126,6 +126,8 @@ def __unicode__(self):
 class UserProfile(models.Model):
     user = models.OneToOneField(User)
     activation_key = models.CharField(max_length=40, blank=True)
+    # default active time is 15 minutes
+    active_time = models.DateTimeField(default=datetime.now()+timedelta(minutes=15))
```

Comment:
I add space around ``+`` now. However, it's suggested not to add space around ``=``when we want to indicate ``keyword``.

https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Whitespace#Whitespace